collectd/java.sls

```diff
@@ -1,22 +1,10 @@
-{% from "collectd/map.jinja" import collectd with context %}
+{% from "collectd/map.jinja" import collectd_settings with context %}
+{% set java_settings = collectd_settings.get('plugins:java') %}
 
 include:
   - collectd
 
-collectd-java:
-    file.rename:
-        - name: {{ collectd.javalib }}
-        - source: {{ collectd.javalib }}.new
-        - force: False
-        - makedirs: False
-
-{{ collectd.javalib }}:
-    file.symlink:
-        - target: {{ salt['pillar.get']('collectd:plugins:java:lib') }}
-        - makedirs: False
-
-
-{{ collectd.plugindirconfig }}/java.conf:
```

Comment:
This is my biggest problem with this PR. The original code seems to be quite broken in that `collectd:plugins:java:lib` isn't always set and this linking does not seem to be necessary on Debian et al. at all.